src/main/java/br/com/tw/lunchandlearn/infrastructure/User.java

```diff
@@ -0,0 +1,14 @@
+package br.com.tw.lunchandlearn.infrastructure;
+
+import org.springframework.data.annotation.Id;
+
+public class User {
+
+    @Id
+    public String id;
+    public String name;
+
+    public User(String name) {
```

Comment:
@pedroaugustofr necessita mesmo desse construtor? Pelo que entendi essa é uma classe de transferencia de dados, então seria interessante ela não ter um comportamento (um construtor, por exemplo)

Reply:
O Construtor foi somente para ter um exemplo de criação de usuario. Se não usar o construtor a criação ficaria assim:
```
User user = new User();
user.name = "Fulano";
```

Com o construtor, pelo menos para mim, me soa mais intuitivo e nos força a criar um usuario que sejá válido.